AniRPG.CharacterSystem.Game.Persistence/AniRPG.CharacterSystem.Game.Persistence.csproj

```diff
@@ -1,10 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
content of this file should be:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
  </PropertyGroup>

</Project>
```
It's very weird. Try to replace current content with suggested.

Reply:
.csproj updated